docs/reference/cli.md

```
@@ -259,49 +259,79 @@ shed image build --from ghcr.io/charliek/shed-fc-base:{version} -n myimage
 
-Lists available image variants from server config and auto-discovered images.
+Lists available image variants from server config plus any tags discovered in the blob store. `shed image list` is a deprecated alias.
 
 ```bash
-shed image list
+shed image ls
 ```
 
 **Output:**
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)